src/hast2jcr/handlers/block.js

```diff
@@ -159,7 +248,7 @@ function generateProperties(node, ctx) {
   }
   const { name, filterId } = findNameFilterById(componentDefinition, id);
   const filter = findFilterById(filters, filterId);
-  const attributes = extractProperties(node, id, componentModels, 'simple');
+  const attributes = extractProperties(node, id, ctx, 'simple');
```

Comment:
the id you pass here should be the model from the componentDefinition you found with `findNameFilterById`